### PR TITLE
Fix kuttl asserts for telemetry changes

### PR DESCRIPTION
**Pull request by cescgina**
Depends-On: https://github.com/openstack-k8s-operators/openstack-operator/pull/349

### CODE REVIEW

File: tests/kuttl/tests/collapsed/01-assert-collapsed-cell.yaml [hunk 1]
```diff
@@ -99,11 +99,10 @@ spec:
...
-  telemetry:
+  ceilometer:
     template:
-      ceilometerCentral:
-        passwordSelector:
-          service: CeilometerPassword
-        secret: osp-secret
-        serviceUser: ceilometer
-      rabbitMqClusterName: rabbitmq
+      serviceAccount: telemetry-operator-telemetry
```
[the remark this answers is not in the document]
Comment (cescgina) [reply tests/kuttl/tests/collapsed/01-assert-collapsed-cell.yaml:104]: Thanks, removed. I think I was looking at an older version of the CR sample when I added them

**Review by abays — APPROVED** [review]
/lgtm

**Comment by abays** [comment]
recheck

**Comment by cescgina** [comment]
recheck

**Comment by cescgina** [comment]
recheck

**Comment by cescgina** [comment]
@abays had to rebase to fix CI, could you please add again the lgtm?

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**